src/env/prod/terraform.tfvars

```diff
@@ -217,7 +217,7 @@ cidr_subnet_eventhub  = ["10.230.6.64/26"]
 devops_service_connection_object_id = "239c15f9-6d56-4b9e-b08d-5f7779446174"
 
 db_sku_name                     = "GP_Gen5_32"
```

Comment:
```suggestion
db_sku_name                     = "GP_Gen5_16"
```

Reply:
it's in here: https://github.com/pagopa/cstar-infrastructure/pull/133